force-app/main/domain/LegacyHouseholds.cls

```diff
@@ -286,9 +286,11 @@ public inherited sharing class LegacyHouseholds extends fflib_SObjects {
         List<Id> householdIdsToUpdate =
                 contactService.moveOppsForContactsSwitchingAccounts(contactsInstance);
         householdService.updateHouseholds(householdIdsToUpdate, dmlWrapper);
+
         Households.updateNameAndMemberCount(
-                contactsInstance.getHouseholdIdsToRename());
-       updateOwners(contactsInstance.getNewOwnerContactById(), dmlWrapper);
+            contactsInstance.getHouseholdIdsForContactsWithDeceasedUpdates(contactsInstance.getHouseholdIdsToRename()));
```

Comment:
I see that this is additive, but the way it's written it feels like it's further filtering the Contact list that's being passed (which we wouldn't want to do). Might just be a code organization thing.

Reply:
Would it help to change the method name to addHouseholdIdsForContactsWithDeceasedUpdates() or do you believe additional code reconfiguration would be a good idea?

Reply:
Forget that suggestion.  Let's get the Ids and the merge in a separate method.  That should clarify what is happening.